### PR TITLE
[THORN-2561] Health 2.2

### PR DESCRIPTION
**Pull request by mirostary**
HealthCheckResponse is a concrete class to allow its usage on consuming end

### CODE REVIEW

File: microprofile/microprofile-health-check-2.2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.10.2.redhat-00003</version>
```
Comment (Ladicek) [inline microprofile/microprofile-health-check-2.2/pom.xml:53]: Why is this dependency even here? We have Gson for the tests, no?

**Comment by Ladicek** [comment]
Otherwise LGTM.

File: microprofile/microprofile-health-check-2.2/pom.xml (hunk 1 above)
Comment (mirostary) [reply microprofile/microprofile-health-check-2.2/pom.xml:53]: Yes, forgot it there.

**Comment by Ladicek** [comment]
Thanks!

**Pull request merged**